packages/inter-protocol/test/vaultFactory/test-prioritizedVaults.js

```diff
@@ -146,36 +139,37 @@ test('update changes ratio', async t => {
     // @ts-expect-error cast
     fakeVault1InitialDebt,
     defaultCollateral,
-    'id-fakeVault1',
+    fakeVaultID1,
   );
   t.is(removedVault, fakeVault1);
-  vaults.addVault('id-fakeVault1', removedVault);
+  vaults.addVault(fakeVaultID1, removedVault);
   // 95n from setDebt / 100n defaultCollateral
   t.deepEqual(vaults.highestRatio(), percent(95));
 
-  const newCollector = makeCollector();
-  rescheduler.resetCalled();
-  Array.from(vaults.entriesPrioritizedGTE(percent(90))).map(
-    newCollector.lookForRatio,
-  );
-  t.deepEqual(
-    newCollector.getPercentages(),
-    [95],
-    'only one is higher than 90%',
+  const crKey = toVaultKey(
+    // @ts-expect-error cast
+    AmountMath.make(brand, 90n),
+    AmountMath.make(brand, 100n),
+    MINIMAL_VAULT_KEY,
   );
-  t.deepEqual(vaults.highestRatio(), percent(95));
-  t.falsy(rescheduler.called(), 'foreach does not trigger rescheduler'); // change from previous implementation
+
+  const vaultsToLiquidate = vaults.removeVaultsBelow(crKey);
```

Comment:
separation of concerns
```suggestion
  const vaultsRemoved = vaults.removeVaultsBelow(crKey);
```

Reply:
sure